src/Foundation/NSOperatingSystemVersion.cs

```diff
@@ -36,20 +37,57 @@ namespace Foundation  {
 	[SupportedOSPlatform ("macos")]
 	[SupportedOSPlatform ("tvos")]
 #endif
-	// NSProcessInfo.h
-	public struct NSOperatingSystemVersion {
+
+	public struct NSOperatingSystemVersion : IEquatable<NSOperatingSystemVersion>, IComparable 
+    {
 		public nint Major, Minor, PatchVersion;
 		
-		public NSOperatingSystemVersion (nint major, nint minor, nint patchVersion)
+        public NSOperatingSystemVersion (nint major, nint minor, nint patchVersion)
```

Comment:
NBD, but this line shows up in the diff even though it seems nothing important changed (I'm guessing it's a whitespace change).

In general we avoid whitespace changes, because it makes diffs harder to read (you end up spending trying to figure out what changed, just to realize nothing did).